### PR TITLE
Adds new integration [Vova-SH/termux-api]

### PR DESCRIPTION
**Pull request by Vova-SH**
## Checklist

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: <https://github.com/Vova-SH/termux-api/releases/tag/v1.0.0>
Link to successful HACS action (without the `ignore` key): <https://github.com/Vova-SH/termux-api/actions/runs/7855954064>
Link to successful hassfest action (if integration): <https://github.com/Vova-SH/termux-api/actions/runs/7855954064>

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -903,6 +903,7 @@
   "viragelabs/virage_dashboard",
   "vlumikero/home-assistant-securitas",
   "vmakeev/huawei_mesh_router",
+  "Vova-SH/termux-api",
```
Comment (ludeeus) [inline integration:906]: PR is not editable.
You need to adjust your branch settings.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (Vova-SH) [reply integration:955]: I created new PR with other source branch: https://github.com/hacs/default/pull/2280
It's ok?

Comment (ludeeus) [reply integration:955]: It is not, it still does not have maintainer permissions.

Comment (Vova-SH) [reply integration:955]: Oh, sorry. I checked "Allow edits by maintainers" now.

**Comment by Vova-SH** [comment]
@ludeeus Can you help with check HACS Action?
This action failed by cause "failed:  required key not provided @ data['issue_tracker'] (More info: https://hacs.xyz/docs/publish/include#check-manifest )".
But in repo has this key was palced that: https://github.com/Vova-SH/termux-api/blob/main/custom_components/termux_api/manifest.json#L10

What is wrong here?

**Comment by ludeeus** [comment]
You are looking at your branch, not your current release.

**Comment by Vova-SH** [comment]
> You are looking at your branch, not your current release.

Thank you so much! Fix it

**Pull request merged**